src/main/java/org/opentripplanner/updater/vehicle_rental/datasources/GbfsGeofencingZoneMapper.java

```diff
@@ -0,0 +1,65 @@
+package org.opentripplanner.updater.vehicle_rental.datasources;
+
+import com.google.common.hash.Hashing;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Objects;
+import org.entur.gbfs.v2_2.geofencing_zones.GBFSGeofencingZones;
+import org.locationtech.jts.geom.Geometry;
+import org.opentripplanner.framework.geometry.GeometryUtils;
+import org.opentripplanner.framework.geometry.UnsupportedGeometryException;
+import org.opentripplanner.routing.vehicle_rental.GeofencingZone;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GbfsGeofencingZoneMapper {
+
+  private static final Logger LOG = LoggerFactory.getLogger(GeofencingZone.class);
+
+  private final String systemId;
+
+  public GbfsGeofencingZoneMapper(String systemId) {
+    this.systemId = systemId;
+  }
+
+  public List<GeofencingZone> mapRentalVehicleType(GBFSGeofencingZones input) {
```

Comment:
Not `mapRentalVehicleType`, but `mapRentalGeofencingZone `

Reply:
Done in a54c0d34c93564d7eca17b3c56c0deb27528177f